src/containers/VisitPage.js

```diff
@@ -374,7 +373,12 @@ class VisitPage extends Component {
         </Container>
 
         <Container id={ `alwaysLeftButtons` }>
-          <ResetAnytime askToResetClient={ this.askToResetClient } />
+          <ButtonReset
+            onClick   = { this.askToResetClient }
+            snippets  = { snippets }
+            overrides = {{ id: `resetFixed`, size: `medium` }}>
+            New Client
```

Comment:
This should be localized into a snippet

Reply:
True. It's new code. It should be snippetized if someone experienced is working on it. Thanks!